### ISSUE TITLE
Generate variable webfonts

### ISSUE THREAD

**Issue by naiyerasif** [JetBrains/JetBrainsMono#519]
Currently, the variable versions are being published only in the `ttf` format. It'd be useful if the `woff2` files can also be added in the distribution so that I can use it as webfont (and potentially avoid downloading multiple font files).

**Comment by jenstroeger** [comment]
[Conversion tools](https://stackoverflow.com/questions/41233532/converting-ttf-to-woff) don’t work?

**Comment by naiyerasif** [comment]
@jenstroeger Unfortunately, they are a hit or miss with variable fonts. I have tried them but they miss certain axes while converting.

**Comment by philippnurullin** [comment]
@RosaWagner Is it possible to add the variable `woff2` to the export script?

**Comment by RosaWagner** [comment]
You can use` fonttools ttLib.woff2` for that (in a bash script also if you want to have it at export, gftools builder doesn't do that for variable fonts)

**Comment by umitseyhan75** [comment]
I also agree that officially provided variable woff2 fonts in the relase zip would be great.

**Comment by naiyerasif** [comment]
@philippnurullin @RosaWagner I took a stab at this; PR here https://github.com/JetBrains/JetBrainsMono/pull/535. I am pretty novice at Python so feel free to nitpick on that change 😄

**Comment by michaeloliverx** [comment]
I would be interested in `woff2` variable fonts too.

**Comment by xqdoo00o** [comment]
would like to use woff2 variable fonts too.

**Comment by philippnurullin** [comment]
Merged the #535. 
The variable .woff2 files will be available after the next push. Will wait until then, and close if everything is fine.

**Comment by lucpotage** [comment]
When downloading JetBrains Mono, I ran into the same issue and looked for the WOFF2 version of the variable font. Do you know when the website package will be updated to reflect this change?
